### PR TITLE
[15.0][ADD]stock_picking_on_hold

### PR DESCRIPTION
**Pull request by dsolanki-initos**

### CODE REVIEW

File: stock_picking_on_hold/models/sale_order.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+from odoo import api, fields, models
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    payment_method_id = fields.Many2one(
```
Comment (rousseldenis) [inline stock_picking_on_hold/models/sale_order.py:7]: IMHO, this has to go in another module

Comment (dsolanki-initos) [reply stock_picking_on_hold/models/sale_order.py:7]: @rousseldenis In which repository we should put this can you please suggest?

Comment (rousseldenis) [reply stock_picking_on_hold/models/sale_order.py:7]: In sale-workflow should be good.

**Comment by rousseldenis** [comment]
@dsolanki-initos As it can be critical flow, could you add some tests ? Thanks

File: stock_picking_on_hold/models/sale_order.py (hunk 1 above)
Comment (dsolanki-initos) [reply stock_picking_on_hold/models/sale_order.py:7]: I've created PR in sale-workflow you can refer this PR :- https://github.com/OCA/sale-workflow/pull/2073

**Pull request closed without merging**